src/rgen/Microsoft.Macios.Generator/Formatters/SmartEnumFormatter.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections.Generic;
```

Comment:
Needs copyright notice

Reply:
AFAIK it does not need to be added. We have a MIT license file that states:

'
The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software
'

My understanding is that all files are already copyrighted using the dotnet foundation copyright + Ms and if and only if someone copies the software or part of it (in this case, this file), they have to add the copyright to it.

Reply:
To add more info, dotnet does not add a copyright comment, in anything they point to the license file:

https://github.com/dotnet/roslyn/blob/main/src/Analyzers/CSharp/Analyzers/AddBraces/CSharpAddBracesDiagnosticAnalyzer.cs

Reply:
You both are right, We do need to add the copyright headers according to our OSS docs https://docs.opensource.microsoft.com/releasing/general-guidance/copyright-headers/ let's not block this PR on this but let's just do a single PR for rgen separately to address this. We can use either copyright header the one in the docs or the one in dotnet.

Reply:
Do we need to add that to all files, we have not done that in a while. What header do we need to use? We should add a template rather than trusting developers to add it